openapi.yaml

```diff
@@ -570,6 +582,30 @@ components:
         - officerImageUrls
         - imageUrls
         - statusRecords
+    ReportStatisticsDto:
+      type: object
+      properties:
+        totalSentReports:
+          type: number
+          format: integer
```

Comment:
The data type should be specified as 'integer' instead of 'number'. For detailed information on data types, refer to the Swagger documentation at https://swagger.io/docs/specification/data-models/data-types/. To address this issue, ensure the correct type is set at the API level. For guidance on types and parameters in NestJS, consult the documentation at https://docs.nestjs.com/openapi/types-and-parameters.